GitVersionConfig.yaml

```diff
@@ -1,3 +0,0 @@
-mode: ContinuousDeployment
-assembly-versioning-scheme: MajorMinorPatch 
-#next-version: 1.0.0
```

Comment:
On second thought, since this project didn't use either one, I think it's fair to say that this PR is an improvement over not using anything at all. So perhaps it's a follow-up change someone else can make.